kodilla-exception/src/main/java/com/kodilla/exception/test/FindAirport.java

```diff
@@ -0,0 +1,41 @@
+package com.kodilla.exception.test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FindAirport {
+
+    public boolean findFlight(Flight flight)throws RouteNotFoundException{
+
+        HashMap<String, Boolean> findAirPort = new HashMap<>();
+
+        findAirPort.put("London", Boolean.TRUE);
+        findAirPort.put("Paris", Boolean.FALSE);
+        findAirPort.put("Berlin", Boolean.TRUE);
+        findAirPort.put("Warszawa", Boolean.TRUE);
+        findAirPort.put("Munich", Boolean.FALSE);
+        findAirPort.put("Frankfurt", Boolean.TRUE);
+        findAirPort.put("Madrid", Boolean.TRUE);
+        findAirPort.put("Roma", Boolean.FALSE);
+        findAirPort.put("Edinburg", Boolean.TRUE);
+        findAirPort.put("NewYork", Boolean.TRUE);
+        findAirPort.put("Chicago", Boolean.FALSE);
+        findAirPort.put("Tokyo", Boolean.TRUE);
+
+        boolean available = false;
+
+        if(findAirPort.containsKey(flight.getArrivalAirport())){
+
+            for(Map.Entry<String, Boolean> entry: findAirPort.entrySet()){
+
+                if (flight.getArrivalAirport().equals(entry.getKey())){
+                    available = entry.getValue();
+                }
+            }
+        }else throw new RouteNotFoundException("Airport not found in map");
+
+
+        return available;
```

Comment:
metoda dziala, ale jest bardzo nieczytelna. Z mapy mozesz wyciagnac wartosc po kluczu, nie ma potrzeby zamiany na entryset. Rzuc okiem na to rozwiazanie
```
        if (findAirPort.containsKey(flight.getArrivalAirport())) {
            return findAirPort.get(flight.getArrivalAirport());
        } else {
            throw new RouteNotFoundException("Airport not found in map");
        }
```